src/lib/widgets/now-playing/ui/now-playing.svelte

```diff
@@ -54,6 +55,7 @@
           />
         {/if}
         <div class="flex flex-col items-center" class:invisible={transcriptShown}>
+          <SwipeDownBar on:minimize={() => nowPlayingActive.set(false)} />
```

Comment:
issue: I think we still want to show the drag indicator even when the transcript is shown